addon/globalPlugins/winWizard.py

```diff
@@ -456,13 +455,10 @@ def terminate(self):
 		del self.hiddenWindowsList
 
 	@scriptHandler.script(
-		description=_(
-			# Translators: Description of the keyboard command
-			# allowing to jump between top-level windows of the current application.
-			"Jumps between top-level windows of the current application."
-		),
+			# Translators: Description of the keyboard command that allowing to jump between top-level windows of the current application.
+		description=_("Jumps between top-level windows of the current application."),
 		gesture="kb:NVDA+windows+Tab",
-		category=SCRCAT_FOCUS,
+#		category=SCRCAT_FOCUS,
```

Comment:
Why this change. I've intentionally placed this script in the system focus category of the input gestures dialog as this is more intuitive that way.